semgrep-core/tests/dockerfile/arg-metavar.dockerfile

```diff
@@ -0,0 +1,7 @@
+# MATCH:
+ARG foo
+
+# MATCH:
+ARG bar
+
+ARG foo=42
```

Comment:
I would have expected a match there too. That's usually how things work for other language where a metavariable can match a var, but also any expression.
Maybe in the context of an ARG it makes sense to restrict a little bit more things. We'll see how user react.